app/src/main/java/org/sopt/dosopttemplate/MainActivity.kt

```diff
@@ -1,11 +1,22 @@
 package org.sopt.dosopttemplate
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import org.sopt.dosopttemplate.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityMainBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        val id = intent.getStringExtra("mainId")
+        val nk = intent.getStringExtra("mainNk")
+        val hm = intent.getStringExtra("mainHm")
+        binding.userId.text = id
+        binding.userNk.text = nk
+        binding.userHome.text = hm
```

Comment:
```suggestion
with(binding){
        userId.text = id
        userNk.text = nk
        userHome.text = hm
}
```
with 스코프 함수를 활용하면 요렇게도 작성 가능합니다!